packages/core/crates/core-p2p/src/lib.rs

```diff
@@ -40,12 +37,17 @@ const HOPR_MESSAGE_REQUEST_TIMEOUT: std::time::Duration = std::time::Duration::f
 const HOPR_ACKNOWLEDGEMENT_CONNECTION_KEEPALIVE: std::time::Duration = std::time::Duration::from_secs(3600); // 1 hour
 const HOPR_ACKNOWLEDGEMENT_REQUEST_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(15);
 
+const HOPR_TICKET_AGGREGATION_CONNECTION_KEEPALIVE: std::time::Duration = std::time::Duration::from_secs(3600); // 1 hour
+const HOPR_TICKET_AGGREGATION_REQUEST_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(15);
```

Comment:
The outer timeout is currently 10 seconds. These should both be configurable.

Reply:
Indeed, waiting for #5486 to be merged.

Reply:
Made this configurable.